automation/generatetasks/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:41
+FROM fedora:42
```

Comment:
42 is not released yet?

Reply:
It is available in dockerhub https://hub.docker.com/_/fedora/tags